src/components/Button/Button.scss

```diff
@@ -112,9 +112,10 @@ $spinner-size: rem(20px);
 // in this case.
 // stylelint-disable selector-max-specificity
 
-.plain {
+.plain,
+.monochrome {
   @include recolor-icon(color('blue'));
-  margin: (-1 * $vertical-padding) rem(-8px);
+  margin: rem(-8px);
```

Comment:
Not sure why we were using `(-1 * $vertical-padding)` because it was misaligning the button text with the surrounding text. Removing it fixed it.

Reply:
I can remove this change if we feel it's too risky. It is only 1 pixel but when inline, like in error text, it's obviously not aligned without this change. I looked at Percy and it looks like it still lines up correctly with cards actions.

Reply:
I’m fine with leaving it in, let’s just keep it in mind in case we see regressions that might be related.